docker_test.go

```diff
@@ -950,7 +950,7 @@ func TestWorkingDir(t *testing.T) {
 func ExampleDockerProvider_CreateContainer() {
 	ctx := context.Background()
 	req := ContainerRequest{
-		Image:        "docker.io/nginx:alpine",
+		Image:        "nginx:alpine",
```

Comment:
suggestion: use the existing constant nginxAlpineImage, more below

Reply:
I did not do it on purpose, as the testable examples could eventually land into a docs page, like pkg.go.dev, and the variable won't be interpolated.